examples/radar_processing_1.ipynb

```diff
@@ -107,8 +116,19 @@
    ],
    "source": [
     "workflow = S1_SNAP_project.create_workflow(name=\"S1-GRD_SNAP\", \n",
-    "                                   use_existing=True)\n",
-    "print(up42.get_blocks(basic=True))"
+    "                                           blocks=['sobloo-s1-grd-fullscene', 'snap-polarimetric'],\n",
+    "                                           use_existing=True)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define aoi and input parameters\n",
+    "\n",
+    "The S1 GRD block always delivers the complete images as they are delivered in SAFE format which cannot be clipped.\n",
+    "The image can be clipped though by the SNAP polarimetric processing blocks. For this it is necessary to supply the same geometry (in this case a bbox) to that block as well. \n",
```

Comment:
Style suggestion: However, the image can be clipped using the SNAP polarimetric processing **block**.